packages/grpc-native-core/binding.gyp

```diff
@@ -194,25 +204,45 @@
               '-Wall',
               '-Wextra',
               '-Werror',
+              '$(W_NO_UNKNOWN_WARNING_OPTION)',
               '-Wno-long-long',
               '-Wno-unused-parameter',
-              '-DOSATOMIC_USE_INLINED=1',
               '-Wno-deprecated-declarations',
-              '-Ithird_party/nanopb',
+              '-Wno-sign-conversion',
+              '-Wno-shadow',
+              '-Wno-conversion',
+              '-Wno-implicit-fallthrough',
+              '-Wno-sign-compare',
+              '-Wno-missing-field-initializers',
+              '-Wno-maybe-uninitialized',
               '-DPB_FIELD_32BIT',
+              '-DOSATOMIC_USE_INLINED=1',
+              '-Ithird_party/nanopb',
+              '-Ithird_party/upb',
+              '-Isrc/core/ext/upb-generated',
               '-fvisibility=hidden',
           ],
           'OTHER_CPLUSPLUSFLAGS': [
               '-g',
               '-Wall',
               '-Wextra',
               '-Werror',
+              '$(W_NO_UNKNOWN_WARNING_OPTION)',
```

Comment:
Is this supposed to be here?

Reply:
Gah, I'm not properly filtering all of them out. Thanks.

Reply:
Fixed.